Mathlib/RingTheory/Trace.lean

```diff
@@ -622,15 +623,15 @@ theorem det_traceForm_ne_zero [IsSeparable K L] [DecidableEq ι] (b : Basis ι K
         simp only [Basis.toMatrix_mul_toMatrix_flip, Matrix.transpose_one, Matrix.mul_one,
           Matrix.det_one]
   simpa only [traceMatrix_of_basis] using det_traceMatrix_ne_zero' pb
+  rw [Basis.toMatrix_mul_toMatrix_flip]
 #align det_trace_form_ne_zero det_traceForm_ne_zero
 
 variable (K L)
 
-theorem traceForm_nondegenerate [FiniteDimensional K L] [IsSeparable K L] :
-    (traceForm K L).Nondegenerate :=
-  BilinForm.nondegenerate_of_det_ne_zero (traceForm K L) _
-    (det_traceForm_ne_zero (FiniteDimensional.finBasis K L))
-#align trace_form_nondegenerate traceForm_nondegenerate
+theorem traceForm_separatingLeft [FiniteDimensional K L] [IsSeparable K L] :
```

Comment:
Why erasing `traceForm_nondegenerate`? Even if `traceForm_separatingLeft` is stronger we can have both (as a lot of people will look for nondegeneracy).